plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/style.scss

```diff
@@ -26,14 +27,14 @@ html body .wp-block-woocommerce-product-details.is-style-minimal {
 			border-bottom-width: 2px;
 			border-radius: 0;
 			margin: 0;
-			padding: 0 1em;
+			padding: 0;
 			display: inline-block;
 			float: none; /* Backward compatibility for TT3, TT2. */
 			font-weight: bold;
 			opacity: 0.65;
 
-			a {
-				padding: 0.5em 0;
+			> a {
```

Comment:
Curious to know the reason for this change?

Reply:
Not a strong opinion, but I feel that when we are sure an element is a [direct child](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) of another, it's better to use `>`, just to prevent the CSS accidentally matching an unwanted element down the tree.

In this case, it shouldn't really matter because we know there aren't any other `a` inside `.wc-tabs li`, so I don't mind removing it if you think that's better.

Reply:
Gotcha, makes sense! I just checked the PHP for this output and there's no filter to manipulate it but there is always the chance of people using JS/jQuery to add icons, change markup etc. It's low risk so I don't mind approving it but if there's no good reason to change whats already working, should we?

Reply:
Sounds good, thanks for the feedback. I removed it in 8b064caa7017c24ee705b9d598bea8750e581013. :slightly_smiling_face: